.github/workflows/jekyll-gh-pages.yml

```
@@ -0,0 +1,66 @@
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-
+progress and latest queued.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)